src/components/molecules/NavSearchBar/NavSearchResult.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import classNames from "classnames";
+
+import "./NavSearchResult.scss";
+
+interface NavSearchResultProps {
+  title: string;
+  description?: string;
+  image?: string;
+  isAvatar?: boolean;
+  onClick?: (e?: React.MouseEvent) => void;
+}
+
+export const NavSearchResult: React.FC<NavSearchResultProps> = ({
+  title,
+  description,
+  image,
+  isAvatar,
+  onClick,
+}) => {
+  return (
+    <div className="nav-dropdown--search-result" onClick={onClick}>
+      <div
+        className={classNames("nav-dropdown--search-result-image", {
+          "nav-dropdown--search-result-avatar": isAvatar,
+        })}
+        style={{
+          backgroundImage: image ? `url(${image})` : "",
+        }}
```

Comment:
Extract this into `const imageStyles =` above rather than inline.

I think if we use `undefined` rather than `""` we will just not end up setting the style, which might be a better case than the empty string.

Reply:
fixed: https://github.com/sparkletown/sparkle/pull/1328/commits/eb24607d724c23e94c3d4d1a40d7df37f6d9489b